georchestra-integration/externalized-accounts/src/main/java/org/geonetwork/security/external/integration/RolesBasedGroupSynchronizer.java

```diff
@@ -71,7 +71,7 @@ public RolesBasedGroupSynchronizer(CanonicalAccountsRepository canonicalAccounts
      */
     public @Override List<CanonicalGroup> fetchCanonicalGroups() {
         List<CanonicalGroup> roles = canonicalAccounts.findAllRoles();
-        Stream<CanonicalGroup> matches = roles.stream().filter(this::matchesRoleNameFilter);
+        Stream<CanonicalGroup> matches = roles.stream().filter(this::notMatchesGeorchestraDefaultRoleNameFilter).filter(this::matchesRoleNameFilter);
```

Comment:
s/notMatches/noMatches/ or doesNotMatch... ?

Reply:
I was tired 😄 doesNotMatch is better, I fix this